src/read.jl

```diff
@@ -104,113 +104,160 @@ function read_storage(filepath)::Storage
 end
 
 
-"""Load demand flexibility profile from .csv files into DataFrame(s)."""
+"""
+    read_demand_flexibility(filepath, interval)
+
+Load demand flexibility profiles and parameters from .csv files and return them in a 
+DemandFlexibility struct.
+"""
 function read_demand_flexibility(filepath, interval)::DemandFlexibility
     # Initialize demand flexibility
     demand_flexibility = Dict()
 
-    # Try loading demand flexibility profiles. Otherwise, create a DataFrame of zeros
+    # Pre-specify the demand flexibility parameters
+    demand_flexibility["enabled"] = true
+    demand_flexibility["duration"] = nothing
+    demand_flexibility["interval_balance"] = true
+    demand_flexibility["rolling_balance"] = true
```

Comment:
Can we do this in one shot, something like
```julia
demand_flexibility = Dict([
   ("enabled", true),
   ("duration", nothing),
   ...
])
```
Also, should we set `demand_flexibility["enabled"] = "not specified"` instead of `true` according to the logic flow?

Reply:
Yeah, I can do both of those things

Reply:
Done.